pages/index.vue

```diff
@@ -54,6 +56,9 @@
     // Allows for calls to store actions
     import { mapActions } from "vuex";
 
+    // Allows for reference to store data by creating simple, implicit getters
```

Comment:
@surchs We no longer need `mapGetters` or `mapMutations` because of the move of `setDataTable` to the store.